src/Orchard/Environment/OrchardStarter.cs

```diff
@@ -174,6 +176,16 @@ public static class OrchardStarter {
             return container;
         }
 
+        /// <summary>
+        /// Get list of comma separated paths from web.config appSettings
+        /// Also return the default path
+        /// </summary>
+        static IEnumerable<string> GetConfigPaths(string key, string defaultPath)
+        {
+            char[] delim = new char[]{','};
+            return (WebConfigurationManager.AppSettings[key] ?? "").Split(delim, StringSplitOptions.RemoveEmptyEntries).Concat(new string[]{defaultPath}).Select(s => s.Trim()).Distinct(StringComparer.OrdinalIgnoreCase);

```

Comment:
Use ConfgurationManager, or better, IAppConfigurationAccessor if possible. The advantage of ConfigurationManager is that it's not ASP.NET-coupled, could be even used when Orchard is run as an exe with [Orchard App Host](http://orchardapphost.codeplex.com/). IAppConfigurationAccessor is of course also flexible in implementation (and by default uses ConfigurationManager)